### ISSUE TITLE
OpenWebRX / SoapyAirspy / 3 gain(s) ?

### ISSUE THREAD

**Issue by zanco** [simonyiszk/openwebrx#94]
Hi, running OpenWebRX with one Airspy mini:

> Using device Airspy: device_id=0 origin=https://github.com/pothosware/SoapyAirspy 
Found 1 antenna(s): RX 
Found 3 gain(s): LNA MIX VGA 
Found 1 frequencies: RF 
Found 2 sample rates: 6000000 3000000 
Found 0 bandwidths: 
[INFO] Using format CS16.
Using output format: CF32 (input format CS16)
Sampling at 6000000 S/s.
Tuned to 438000000 Hz.
Tuner gain set to 12.00 dB.

How do I pass the 3 gain values to the Airspy Mini ? 

Thanks,
Ben

**Comment by ha7ilm** [comment]
Yes, that's a problem. `rx_tools` currently does not have a generic way to set gain values:

https://github.com/rxseger/rx_tools/blob/master/src/convenience/convenience.c#L262

...it is rather hardwired for HackRF and RTL-SDR.
